selfhost/types.jakt

```diff
@@ -1038,6 +1046,7 @@ boxed enum CheckedExpression {
 
     function control_flow(this) -> BlockControlFlow => match this {
         Match(expr, match_cases, span, type_id, all_variants_constant) => {
+            // eprintln("control flow for expr: {}", expr)
```

Comment:
This looks like it shouldn't be here. :^)

Reply:
good catch!